.gitignore

```diff
@@ -42,6 +43,12 @@ modules/puppet_agent
 modules/puppet_conf
 modules/service
 
+<<<<<<< HEAD
+# OS generated files #
+######################
+.DS_Store
```

Comment:
This is better to put in `~/.gitignore_global`, since it's OS-specific and will show up pretty much everywhere.